tests/conftest.py

```diff
@@ -16,6 +16,8 @@
 from topostats.tracing.dnatracing import dnaTrace, traceStats
 from topostats.utils import get_thresholds, get_mask, _get_mask
 
+from pySPM import SPM
```

Comment:
Pylint says this isn't used within `conftest.py` and indeed it is only imported...

```
❱ grep SPM tests/conftest.py -n
19:from pySPM import SPM
```

Reply:
I think I was trying to use it for a typehint when trying to make a test for the spm px_to_nm function.

I'll remove it now.